src/main/java/io/github/dft/woocommerce/model/orderapi/ordernote/OrderNoteWrapper.java

```diff
@@ -0,0 +1,11 @@
+package io.github.dft.woocommerce.model.orderapi.ordernote;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Data;
+
+import java.util.ArrayList;
+
+@Data
```

Comment:
What is the use of @Data here have you checked without @Data ?

Reply:
No, i am checking that.

Reply:
check it and let me know the result.

Reply:
Yes, it is working without @Data.